Update readme (#24)
dwave-examples/feature-selection-cqm: README.md

relationships to the target variable. Correlation between features generates the 
redundancy metric, which is minimized between each pair of features to promote 
diversity. Correlation between features and the target is maximized to promote a 
strong relationship. 

The constraints here focus on choosing the exact number of defined features.

## Model Overview

In this example we use the Titanic and Scene datasets to generate a Constrained Quadratic Model. 
The datasets are assumed to be clean, meaning there are no missing entries or repeated features. 
The features of the dataset are used to build a correlation matrix which compares the features to 
each other as well as a correlation matrix that compares the features to the target variable. Those 
correlation matrices are used to build the objective function. 

---
**Note:** Although a model overview is provided here, all of the code to build
the feature selection model is contained within 
[D-Wave's scikit-learn plug-in](https://github.com/dwavesystems/dwave-scikit-learn-plugin).
---

### Parameters

These are the parameters of the problem:

- `num_features`: the number of selected features

> - `redund_val`: used to determine factor applied to redundancy terms

Reviewer: I think this needs more description. From a quick look I think this works as follows: it's a parameter between 0 and 1 that controls the relative weight of
what you called the correlation and redundancy terms. 

* 0: features will be selected as to minimize the  redundancy without any consideration to quality.
 * 1:  places the maximum weight on the quality of the features

In our code it comes as ``data.solve_feature_selection(num_features, 1.0 - redund_value)``

Author: Ok, I can add a bit more, I was trying to avoid describing the 1.0-redund_value because I think that would be distracting for the reader, but the bulleted suggestions make that easier. Will add that in another commit.

### Variables
- `x_i`: binary variable that shows if feature `i` is selected

> ### Expressions
>  Expressions are linear or quadratic combinations of variables used for easier representations of the models. 
> - None

Reviewer: ```suggestion

```

I think if we have nothing to say in a subsection, better not to have that subsection

### Objective

> The objective function has two terms. The first term corresponds to minimizing the correlation between 
> chosen features in the dataset controlled by the redundancy parameter. The second term corresponds to 
> maximizing the correlation between the features and the target variable. 

Reviewer: ```suggestion
The objective function has two terms. The first term minimizes the correlation between 
chosen features in the dataset (this term is weighted by the redundancy parameter, `redund_val`). The 
second term maximizes the correlation between the features and the target variable. 
```

### Constraints

> A single constraint is used to enforce the number of features selected by the model matches the `num_features`
> parameter.

Reviewer: ```suggestion
A single constraint is used to require that the model select a number of features equal to the `num_features`
parameter.
```

To stay with your phrasing you need to add a "that" or switch to something like " enforce a match between the number of features selected by the model and the `num_features` parameter"
